docs: clarify autosaving and versioned flow saving (#12794)
langflow-ai/langflow: docs/docs/Flows/concepts-flows.mdx

4. Select a previously saved version to preview the flow in read-only mode.
5. To replace the current draft with the selected version, click **Restore**.

From a saved version's <Icon name="EllipsesVertical" aria-hidden="true" /> menu, you can also **Export** or **Delete** that version.

Saved flow versions are stored in the database configured for your Langflow deployment.

> When restoring a saved version, keep **Save current draft before restoring** enabled to create a backup of your current work before the selected version replaces it.

Reviewer: ```suggestion
When restoring a saved version, if **Save current draft before restoring** is enabled, then Langflow creates a backup of your current work before the selected version replaces it.
```
